### ISSUE TITLE
missing book?

### ISSUE THREAD

**Issue by maxlath** [frimelle/guess-the-quote#1]
could that be that the actual book the quote comes from is missing from the offered choices?
I'm quite sure this one comes from a Harry Potter (somewhere between 4th and 6th I would say ^^)
![error](https://cloud.githubusercontent.com/assets/1596934/10528481/ca8c02a6-7396-11e5-89ca-403a9469bca8.png)

**Comment by frimelle** [comment]
Yes, there is something completly messed up with the resolving of quote -> book. I'll look into that!

**Comment by frimelle** [comment]
Might be fixed now. Mind having a quick look?
